### PR TITLE
Feature external site

### PR DESCRIPTION
**Pull request by kalomak**

### CODE REVIEW

File: client/src/components/comment-panel/comment-panel.js [hunk 1]
```diff
@@ -123,9 +125,15 @@ class CommentPanel extends React.Component {
     if (el) el.scrollTop = el.scrollHeight
   }
 
+  handleRouteContainerClick() {
+
```
Comment (kalomak) [inline client/src/components/comment-panel/comment-panel.js]: ?

File: client/src/create.js [hunk 2]
```diff
@@ -36,20 +43,166 @@ class Create extends React.Component {
...
+    event.nativeEvent.stopImmediatePropagation()
+    const doc = shadowDocument()
+    const inputValue = name => doc.getElementById(name).value
+
+    const json = await apiCall('POST', '/instances/new', {
+      url: inputValue('url'),
+      name: inputValue('name').toLowerCase(),
+      type: 'site',
     })
+    window.location.replace(`//${json.containerInfo.subdomain}.localhost:8080`)
```
Comment (kalomak) [inline client/src/create.js]: fix link

Comment (bqqbarbhg) [reply client/src/create.js]: You can use `window.location` for the domain

**Review by HenrikHelenius — DISMISSED** [review]
ok

File: server/src/routes/instances.js [hunk 3]
```diff
@@ -65,12 +59,14 @@ router.post('/new', catchErrors(async (req, res) => {
     url, version, type, name, port,
   } = req.body
 
+  console.log('dddd')
```
Comment (bqqbarbhg) [inline server/src/routes/instances.js]: Debug logging

File: server/src/routes/instances.js [hunk 4]
```diff
@@ -65,12 +59,14 @@ router.post('/new', catchErrors(async (req, res) => {
...
   const { userId } = await reqUser(req)
 
+  console.log('ffff')
```
Comment (bqqbarbhg) [inline server/src/routes/instances.js]: Here too

**Review by bqqbarbhg — DISMISSED** [review]

File: server/src/routes/instances.js (hunk 3 above)
Comment (kalomak) [reply server/src/routes/instances.js]: 😰

File: .gitignore [hunk 5]
```diff
@@ -22,3 +22,6 @@ dev_db.sqlite
 
 # Proxy on mac and linux
 proxy/proxy
+
+# Vim binaries on mac
+*.swp
```
Comment (joonalillfors) [inline .gitignore:27]: Approved

**Review by joonalillfors — APPROVED** [review]
ship it

**Pull request merged**